pallets/funding/src/lib.rs

```diff
@@ -18,96 +18,51 @@
-//!
-//! ## Credentials
-//! The pallet will only allow users with certain credential types, to execute certain extrinsics.:
-//!
-//!
-//! | Extrinsic                                     | Issuer | Retail Investor | Professional Investor | Institutional Investor |
-//! |-----------------------------------------------|--------|-----------------|-----------------------|------------------------|
-//! | `create`                                      | X      |                 |                       |                        |
-//! | `edit_metadata`                               | X      |                 |                       |                        |
-//! | `start_evaluation`                            | X      |                 |                       |                        |
-//! | `start_auction`                               | X      |                 |                       |                        |
-//! | `bond_evaluation`                             |        | X               | X                     | X                      |
-//! | `failed_evaluation_unbond_for`                |        | X               | X                     | X                      |
-//! | `bid`                                         |        |                 | X                     | X                      |
-//! | `contribute`                                  |        | X               | X*                    | X*                     |
-//! | `vested_plmc_bid_unbond_for`                  |        |                 | X                     | X                      |
-//! | `vested_plmc_purchase_unbond_for`             |        | X               | X                     | X                      |
-//! | `vested_contribution_token_bid_mint_for`      |        |                 | X                     | X                      |
-//! | `vested_contribution_token_purchase_mint_for` |        | X               | X                     | X                      |
-//!
-//! _* They can call contribute only if the project is on the remainder round._
-//!
+//! 1) **Project Creation**: Issuer creates a project with the [`create_project`](Pallet::create_project) extrinsic.
+//! 2) **Evaluation Start**: Issuer starts the evaluation round with the [`start_evaluation`](Pallet::start_evaluation) extrinsic.
+//! 3) **Evaluate**: Evaluators bond PLMC on the project with the [`bond_evaluation`](Pallet::evaluate) extrinsic.
+//! 4) **Evaluation End**: Anyone can end the evaluation round with the [`end_evaluation`](Pallet::end_evaluation) extrinsic after the defined end block.
+//! 5) **Auction Start**: If the project got >=10% of its requested funding (USD) in PLMC bonded, the auction starts immediately after `evaluation_end` was called.
+//! 6) **Bid**: Professional and Institutional investors can bid on the project with the [`bid`](Pallet::bid) extrinsic. Price starts at the issuer defined minimum price, and goes up in a price delta and bucket size of 10%
```

Comment:
**Bid**: Professional and institutional participants can place bids on the project using the [`bid`](Pallet::bid) extrinsic. The price starts at the issuer-defined minimum and increases by increments of 10% in price and bucket size.